hack/package/deb/build.sh

```diff
@@ -1,35 +1,19 @@
 #!/usr/bin/env sh
 
-set -e
-# This script is to build pouch rpm package as follows,
-# Following the below command to build rpm
-# 1. Build pouch:deb image
-#	cd pouch
-#	docker build -t pouch:deb -f hack/package/deb/ubuntu-xenial/Dockerfile.x86_64 .
-# 2. Mount a directory which contains gpg keys, eg
-#    $ tree /root/deb/
-#		deb
-#		├── keys
-#		│   ├── gpg
-#		│   └── secretkey
-# 3. run pouch:deb to generate deb package
-#	 docker run -it -e VERSION="0.1.0" -v /root/deb/:/root/deb pouch:deb bash -c hack/package/deb/build.sh
-# 4. In this example, rpm package will be output in '/root/deb/package/' directory
-
-DIR="$( cd "$( dirname "$0" )" && pwd )"
+set -ex
 
 TMP=$(mktemp -d /tmp/pouch.XXXXXX)
 
 MOUNTDIR=/root/deb
-PACKAGEDIR=/root/deb/package
+PACKAGEDIR=/root/deb/package/deb/
 BASEDIR=/go/src/github.com/alibaba/pouch
 
-rm -rf $BASEDIR/debian && cp -r $DIR/debian $BASEDIR/
-rm -rf $BASEDIR/systemd && cp -r $DIR/systemd $BASEDIR/
+rm -rf $BASEDIR/debian && cp -r $BASEDIR/hack/package/deb/debian $BASEDIR/
+rm -rf $BASEDIR/systemd && cp -r $BASEDIR/hack/package/deb/systemd $BASEDIR/
 
 if [ ! -e $$BASEDIR/Makefile.bak ] ;then
 	mv $BASEDIR/Makefile $BASEDIR/Makefile.bak
-	cp $DIR/Makefile $BASEDIR/Makefile
+	cp $BASEDIR/hack/package/deb/Makefile $BASEDIR/Makefile
```

Comment:
Maybe the `cp` option could put outside `if` scope, as even if there is a Makefile.bak, it doesn't mean the Makefile is which we want.